Makefile

```
@@ -231,7 +270,7 @@ reset-db: ## Reset the PostgreSQL database
 	$(COMPOSE) stop
 	$(COMPOSE) down postgresql metabase
 	$(MAKE) migrate-api
-	$(MAKE) create-superuser
+	$(MAKE) create-api-superuser
 	$(MAKE) create-api-test-db
 	$(MAKE) create-metabase-db
```

You also need to create the Dashboard db, run migrations and create the super user !

```suggestion
	$(MAKE) create-metabase-db
	$(MAKE) create-dashboard-db
	$(MAKE) migrate-dashboard-db
	$(MAKE) create-dashboard-superuser
```